interface/reports/front_receipts_report.php

```diff
@@ -114,6 +115,48 @@ function show_receipt(pid,timestamp) {
 
     <table class='text'>
         <tr>
+            <td class='control-label'>
+                <?php echo xlt('Facility'); ?>:
+            </td>
+            <td>
+                <?php
+                $form_facility=$_REQUEST['form_facility']; 
```

Comment:
why using REQUEST rather than POST here and below?

Reply:
Oops, I focused on the feature, so that I missed the common standards used already in OpenEMR development. Will fix and push the code again.